app/Models/Extensions/PhotoCast.php

```diff
@@ -15,6 +16,40 @@ trait PhotoCast
 	 */
 	public function toReturnArray(): array
 	{
+		if (strpos($this->type, 'video') === 0) {
+			$filename = $this->thumbUrl;
+		} elseif ($this->type == 'raw') {
+			// It's a raw file -> we also use jpeg as extension
+			$filename = $this->thumbUrl;
+		} else {
+			$filename = $this->url;
+		}
+		$filename2x = ($filename !== '') ? Helpers::ex2x($filename) : '';
+		$thumbFileName2x = $this->thumb2x === '1' ? Helpers::ex2x($this->thumbUrl) : null;
+
+		// The original size is not stored in this sub-array but on the root level of the JSON response
+		// TODO: Maybe harmonize and put original variant into this array, too? This would also avoid an ugly if branch in SymLink#override.
+		$sizeVariants = [
+			Photo::VARIANT_THUMB => $this->serializeSizeVariant(
+				Photo::VARIANT_THUMB, $this->thumbUrl, Photo::THUMBNAIL_DIM, Photo::THUMBNAIL_DIM
+			),
+			Photo::VARIANT_THUMB2X => $this->serializeSizeVariant(
+				Photo::VARIANT_THUMB, $thumbFileName2x, Photo::THUMBNAIL2X_DIM, Photo::THUMBNAIL2X_DIM
```

Comment:
```suggestion
				Photo::VARIANT_THUMB2x, $thumbFileName2x, Photo::THUMBNAIL2X_DIM, Photo::THUMBNAIL2X_DIM
```

Reply:
Damn. Silly copy&paste error. A fix is on the way.

Reply:
You can just use the "commit suggestion" button. :D

Reply:
After I did, the automatic tests failed. I'll have to undo this "commit suggestion" and investigate more thoroughly. Actually it should not fail.

Reply:
Problem solved. It was a simply case problem in the name of the constant.